### PR TITLE
Guard against null config

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: src/containers/calculation.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ function mapStateToProps(state, ownProps) {
   }
 
   const config = selectors.configuration.getConfiguration(state);
-  if (isNil(config.features)) {
+  if (!isNil(config.features) && !isNil(config.features)) {
```
Comment (cryos) [inline src/containers/calculation.js:259]: I don't understand, why is this necessary? The same statement is repeated twice and && does nothing as both times it should evaluate to the same thing shouldn't it?

Comment (cjh1) [reply src/containers/calculation.js:259]: That is was it known as a typo

Comment (cryos) [reply src/containers/calculation.js:259]: Makes sense, otherwise I was at a total loss, the new patch makes perfect sense :+1:
